workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+---
+name: 'Reading the Internet'
+description: 'Use Python to read HTML and extract any information you can find!'
+author: '@RafaelCenzano'
+img: 'https://cloud-p5pdve0kf.vercel.app/2webscrape.jpg'
+---
+
+We can read all the content on websites, but wouldn't it be cool if we could automatically read specific data from websites whenever you wanted? If this interested you, this workshop is for you. I will be showing you how to use webscraping in Python.
+
+*Note: not all webpages can be read using this technique. If data is loaded in using ajax or after the inital page request this method may not work.*
+
+[Final Result and Code](https://repl.it/@SavageCoder77/Read-The-Internet#main.py)
+
+## Prerequisites
+You should have a basic understanding of **HTML** and **Python** and how to use variables and objects.
+
+## Setup
+![Lets do it gif](https://cloud-p23nck0rj.vercel.app/1lets-start.gif)
+
+We will be using [repl.it](https://repl.it), an online code editor that allows you to write and run code from any computer!
+
+Spin up a Python3 repl by [clicking here](https://repl.it/languages/python3).
+
+Before we can start coding, we need to install a couple of external libraries. Follow the image below to find where to add the packages. Once in this menu you will need to install `beautifulsoup4`, `requests`, and `lxml`.
+
+![Repl.it packages menu](https://cloud-p5pdve0kf.vercel.app/0ezgif.com-video-to-gif.gif)
+
+#### What each package does
+`lxml` is a C package used by beautiful soup that parses the HTML, `beautifulsoup4` is for working with the parsed HTML, and `requests` is for requesting the webpage/html.
```

Comment:
This is a good simple explanation for how each library applies to this workshop, but I think you should also include a little more information about what each library is. I still don't fully understand exactly what BeautifulSoup is—"working with the parsed HTML" doesn't mean much to me. Similar to `requests`: requests is a library that makes it super easy to make HTTP requests in Python, and in this workshop we're going to use it to "request" the HTML of a website.

You should make it super super clear, almost overexplaining what each library does, because understanding this is essential to understanding the rest of the workshop, and in its current state most club members will likely get lost if they don't understand what these libraries do.